packages/sign_in_with_apple/README.md

```diff
@@ -162,4 +162,19 @@ The `PARAMETERS FROM CALLBACK BODY` should be filled with the urlencoded body yo
 
 Furthermore, when handling the incoming credentials on the client, make sure to only overwrite the current (guest) session of the user once your own server have validated the incoming `code` parameter, such that your app is not susceptible to malicious incoming links (e.g. logging out the current user).
 
-### iOS
\ No newline at end of file
+### iOS
+
```

Comment:
```suggestion
At this point, you should have added the Sign in with Apple capability to either your own app's capabilities or the test application you created to run the example.

In case you don't have `Automatically manage Signing` turned on in XCode, you will need to recreate your Provisioning Profiles for your App, so they include the new `Sign in with Apple` capability. Then you can download the new certificates and select them in XCode.
In case XCode manages your signing, this step will be done automatically for you.

Additionally, this assumes that you have at least one iOS device registered in your developer account for local testing.
```

What do you think about this? I think it's more friendly to users who don't manage their certificates through XCode.